package.json

```diff
@@ -1,83 +1,109 @@
 {
   "name": "react-boilerplate",
-  "version": "1.0.0",
-  "description": "Boilerplate for our React applications",
-  "main": "src/index.js",
-  "author": "Christoffer Niska <christofferniska@gmail.com>",
-  "license": "MIT",
+  "version": "2.0.0",
+  "private": true,
+  "dependencies": {
+    "normalize.css": "^6.0.0",
+    "react": "^15.5.4",
+    "react-dom": "^15.5.4",
+    "react-redux": "^5.0.4",
+    "react-router": "3",
+    "redux": "^3.6.0",
+    "redux-actions": "^2.0.2",
+    "redux-saga": "^0.14.7"
+  },
+  "devDependencies": {
```

Comment:
We've discussed this once already. The fact is that there are no "production dependencies" since the production build is just a bunch of `.js` files and some assets.

This means that in order to build the production bundle you have to install every single dependency, including `webpack-dev-server`.

You have to ask yourself "what can I do if I run `yarn install --production`"? With the current `package.json` you will not be able to do anything, not even, i.e. none of the commands here will work (AFAIK):

```
+  "scripts": {
+    "ci": "CI=true run-p flow lint-staged test",
+    "flow": "flow",
+    "lint": "eslint src --fix",
+    "lint-staged": "lint-staged",
+    "precommit": "lint-staged",
+    "stylelint": "stylelint 'src/**/*.css'",
+    "start": "node scripts/start.js",
+    "build": "node scripts/build.js",
+    "test": "node scripts/test.js --env=jsdom"
+  },
```